File: src/main/java/net/atos/zac/app/zaken/model/RESTZaakOntkoppelGegevens.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.zaken.model;

import java.util.UUID;

public class RESTZaakOntkoppelGegevens {

    public UUID zaakUuid;

    public String gekoppeldeZaakIdentificatie;

    public RelatieType relatietype;

    public String reden;
}

File: src/main/java/net/atos/zac/app/zaken/model/RESTZaakOntkoppelGegevens.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.zaken.model

import nl.lifely.zac.util.AllOpen
import nl.lifely.zac.util.NoArgConstructor
import java.util.*

@AllOpen
@NoArgConstructor
data class RESTZaakOntkoppelGegevens(
    var zaakUuid: UUID,

    var gekoppeldeZaakIdentificatie: String,

    var relatietype: RelatieType,

    var reden: String
)
